python/pyspark/sql/functions/builtin.py

```diff
@@ -16303,7 +16303,21 @@ def from_xml(
     >>> df.select(sf.from_xml(df.value, schema).alias("xml")).collect()
     [Row(xml=Row(a=1))]
 
-    Example 2: Parsing XML with :class:`ArrayType` in schema
+    Example 2: Parsing XML with a :class:`StructType` schema
```

Comment:
Also, can you please enable tests [here](https://github.com/apache/spark/blob/master/python/pyspark/sql/tests/connect/test_connect_function.py#L1911-L1915)

Reply:
sure